content/authors/lt-col-theodore-j-erickson/_index.md

```diff
@@ -12,7 +12,7 @@ last_name: "Erickson"
 pronoun: ""
 
 # Email — If you include an email address, it will be displayed on your profile page
-email: theodore.erickson@us.af.mil
```

Comment:
The spreadsheet lists this as `Federal` [here](https://docs.google.com/spreadsheets/d/1NCE1GqifPoY6t-xZDej8MPESW010lie4gTQGzLk8K6I/edit#gid=0)